packages/contract-addresses/addresses.json

```diff
@@ -460,5 +460,89 @@
             "fillQuoteTransformer": "0xa825d4d3c4d2820c52da69fcccf269b4081871f2",
             "positiveSlippageFeeTransformer": "0x9ffc7a79133ed5242777e40764777a6d5aab282c"
         }
+    },
+    "10": {
+        "erc20Proxy": "0x0000000000000000000000000000000000000000",
+        "erc721Proxy": "0x0000000000000000000000000000000000000000",
+        "zrxToken": "0x0000000000000000000000000000000000000000",
+        "etherToken": "0x4200000000000000000000000000000000000006",
+        "exchangeV2": "0x0000000000000000000000000000000000000000",
+        "exchange": "0x0000000000000000000000000000000000000000",
+        "assetProxyOwner": "0x0000000000000000000000000000000000000000",
+        "zeroExGovernor": "0x0000000000000000000000000000000000000000",
+        "forwarder": "0x0000000000000000000000000000000000000000",
+        "coordinatorRegistry": "0x0000000000000000000000000000000000000000",
+        "coordinator": "0x0000000000000000000000000000000000000000",
+        "multiAssetProxy": "0x0000000000000000000000000000000000000000",
+        "staticCallProxy": "0x0000000000000000000000000000000000000000",
+        "erc1155Proxy": "0x0000000000000000000000000000000000000000",
+        "devUtils": "0x0000000000000000000000000000000000000000",
+        "zrxVault": "0x0000000000000000000000000000000000000000",
+        "staking": "0x0000000000000000000000000000000000000000",
+        "stakingProxy": "0x0000000000000000000000000000000000000000",
+        "erc20BridgeProxy": "0x0000000000000000000000000000000000000000",
+        "erc20BridgeSampler": "0x0000000000000000000000000000000000000000",
+        "chaiBridge": "0x0000000000000000000000000000000000000000",
+        "dydxBridge": "0x0000000000000000000000000000000000000000",
+        "godsUnchainedValidator": "0x0000000000000000000000000000000000000000",
+        "broker": "0x0000000000000000000000000000000000000000",
+        "chainlinkStopLimit": "0x0000000000000000000000000000000000000000",
+        "maximumGasPrice": "0x0000000000000000000000000000000000000000",
+        "dexForwarderBridge": "0x0000000000000000000000000000000000000000",
+        "exchangeProxyGovernor": "0x6d506b2847df0c6f04d2628da1adaf4d8fb2e81b",
+        "exchangeProxy": "0xdef1abe32c034e558cdd535791643c58a13acc10",
```

Comment:
EP verification on Optimistic Etherscan looks a bit funky https://optimistic.etherscan.io/address/0xdef1abe32c034e558cdd535791643c58a13acc10#code. Can we still verify it properly?